src/System.IO.Ports/tests/SerialPort/DosDevices.cs

```diff
@@ -139,7 +139,7 @@ private static char[] CallQueryDosDevice(string name, out int dataSize)
         public const int ERROR_INSUFFICIENT_BUFFER = 122;
         public const int ERROR_MORE_DATA = 234;
 
-        [DllImport("Kernel32.dll", SetLastError = true, CharSet = CharSet.Auto)]
+        [DllImport("Kernel32.dll", SetLastError = true, EntryPoint = "QueryDosDeviceW", CharSet = CharSet.Unicode)]
```

Comment:
The A/W probing doesn't happen on .NET Native. This is exactly was causing Ports to die on startup.